package/gluon-web-autoupdater/luasrc/lib/gluon/web/model/admin/autoupdater.lua

```diff
@@ -32,6 +32,31 @@ function o:write(data)
 	uci:set("autoupdater", autoupdater, "branch", data)
 end
 
+o = s:option(Value, "minute", translate("Minute"), translate(
+	"This value forces the autoupdater to check for updates at the "
+	.. "specified minute. Normaly there is no need to set this value, "
+	.. "because it is selcted automatically. You may wanna set this to a "
+	.. "specific value, if you have multiple nodes which schould not "
```

Comment:
The above lines should not have commas in English.
Typos:
- Normally
- selected
- want to
- should